app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="button_default_style">
```

Comment:
do not create a style for only one property

Reply:
Done